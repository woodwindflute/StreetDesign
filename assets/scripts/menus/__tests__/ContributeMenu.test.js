/* eslint-env jest */
import React from 'react'
import { render } from '../../../../test/helpers/render'
import ContributeMenu from '../ContributeMenu'

describe('ContributeMenu', () => {
  it('renders', () => {
    const { asFragment } = render(<ContributeMenu isActive={true} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
