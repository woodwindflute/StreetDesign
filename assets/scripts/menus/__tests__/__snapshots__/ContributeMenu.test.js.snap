// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContributeMenu renders 1`] = `
<DocumentFragment>
  <div
    class="menu"
  >
    <a
      href="https://opencollective.com/streetmix/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Donate
    </a>
  </div>
</DocumentFragment>
`;
